### ISSUE TITLE
Behöver ett sätt för par att byta kod mellan grupper

### ISSUE THREAD

**Issue by EliasC** [IOOPM-UU/ioopm15#2]
I fas 1 sprint 3 kommer man behöva byta kod med ett annat par, och det är inte säkert att det kommer finnas ett matchande par inom samma grupp. Det vore trevligt med en GitHub-lösning för att komma åt varandras repon, även om man tillhör olika team.

**Comment by phroggyy** [comment]
Finns det några nackdelar med följande struktur:

```
Team {
    par1 {
        "person1",
        "person2",
    },
    par2 {
        "person3",
        "person4",
    },
    ...
}
```

Dvs alla gruppmedlemmar är med i sin grupps team, men har också ett eget team för sitt par. På det sättet skulle ett annat par från en annan grupp kunna få tillgång till ett särskilt repo.

**Comment by EliasC** [comment]
@phroggyy Jag håller med om att det vore en smidig lösning, men det blir en hel del administration om 130 studenter dynamiskt ska kunna bilda småteam med varandra. Kanske går det att scripta något som skapar dessa team, och så har man anmälning på samma sätt som det funkar med att skaffa privata repon just nu.

**Comment by phroggyy** [comment]
@EliasC Det går att använda [GitHubs API](https://developer.github.com/v3/orgs/teams/) för att skapa teams. Bör gå rätt snabbt att bygga en team-manager så länge man har strukturerad data om vem som jobbar med vem.

**Comment by EliasC** [comment]
@phroggyy Det är vad vi använder för att bjuda in folk till organisationen just nu! Det administrativa extraarbetet kommer om varje par får välja själv vem de ska dela kod med (alternativet är förstås extraarbete i implementation av någon slags team-manager). Fanns det en GitHub-lösning (typ om man kunde bjuda in folk från andra teams att läsa ens eget repo) så skulle vi slippa extraarbetet helt :)

**Comment by phroggyy** [comment]
@EliasC Right... Hmm, det går ju att lägga till ett team på ett repo, men det förutsätter ju att man har ett team per par då. På vanliga repos kan man ju bjuda in contributors, men vet inte hur det funkar inom organisationer.
